core/quantity/src/Parser.ts

```diff
@@ -391,6 +391,22 @@ export class Parser {
         tokens.push(new ParseToken(wipToken));
       }
     }
+    if (tokens.length === 1 && typeof tokens[0].value === "number" && format.type === "Bearing") {
```

Comment:
this is a nitpick from me, but you can make the `format.type === "Bearing"` check be first in the if statement, that way if it's not a bearing the if statement won't bother checking the rest of the other segments and just return false